src/components/videos-studio/scroll-up-button/index.tsx

```diff
@@ -0,0 +1,55 @@
+import { useEffect, useState } from "react";
+import { styled } from "@mui/system";
+import NorthIcon from "@mui/icons-material/North";
+const ScrollToTopButton = () => {
+    const [scrollUpButtonIsVisible, setScrollUpButtonIsVisible] =
+        useState(false);
+
+    useEffect(() => {
+        const handleScrollTop = () => {
+            if (
+                document.body.scrollTop > 20 ||
+                document.documentElement.scrollTop > 20
+            ) {
+                setScrollUpButtonIsVisible(true);
+            } else {
+                setScrollUpButtonIsVisible(false);
+            }
+        };
+
+        window.addEventListener("scroll", handleScrollTop);
+
+        return () => {
+            window.removeEventListener("scroll", handleScrollTop);
+        };
+    }, []);
+
+    const scrollToTop = () => {
+        document.body.scrollTop = 0;
+        document.documentElement.scrollTop = 0;
+    };
+
+    const StyledButton = styled("button")({
+        position: "fixed",
+        bottom: "20px",
+        right: "20px",
+        backgroundColor: "#fff",
+        color: "#2d2d2d",
+        border: "none",
+        borderRadius: "5px",
+        padding: "10px",
+        cursor: "pointer",
+        height: "60px",
+    });
+
+    return (
+        <StyledButton
+            onClick={scrollToTop}
+            style={{ display: scrollUpButtonIsVisible ? "block" : "none" }}
+        >
```

Comment:
Instead of using inline styles, consider using CSS classes or Material-UI's sx prop for styling. This approach is more consistent with Material-UI's design system and can make your code easier to manage and understand.